azure-resources/Network/virtualNetworkGateways/recommendations.yaml

```diff
@@ -68,7 +68,7 @@
 
 - description: Configure customer-controlled ExpressRoute gateway maintenance
   aprlGuid: 3e115044-a3aa-433e-be01-ce17d67e50da
-  recommendationTypeId: null
+  recommendationTypeId: B4AF9E04-3570-41F1-B4CF-B7AF07224799
```

Comment:
According to the validator action, GUID should be in lower case.

> 4.recommendationTypeId: 'B4AF9E04-3570-41F1-B4CF-B7AF07224799' is not a regex match.

https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/blob/8c1230c67df9262e131f3d5986f80912bfa2954b/.github/scripts/schemas/aprl-schema.yaml#L7